src/components/ContinuationIn/ExtraproBusinessLookup.vue

```diff
@@ -91,6 +91,10 @@ export default class ExtraproBusinessLookup extends Vue {
 
   readonly businessLookupLabel = 'Extraprovincial registration number or name of the business in B.C.'
 
+  get showBusinessSearchStatus (): boolean {
+    return Boolean(GetFeatureFlag('show-business-search-status'))
```

Comment:
In other places, we were doing `!!GetFeatureFlag(xxx)`. Same thing though right?

Reply:
Yes, same thing. This may be more type-friendly and the other way is kind of a shortcut. I have no preference either way.